### ISSUE TITLE
Github Action to automatically create an image and push it to artifact registry upon every push

### ISSUE THREAD

**Issue by DonggeLiu** [google/oss-fuzz-gen#67]

**Comment by oliverchang** [comment]
Would a GCB trigger be the better approach here? The auth story would be a likely be a lot clearer there.

**Comment by erfanio** [comment]
Good point about the auth. GCB trigger is probably better for pushes to main branch.

Another thing Dongge and I discussed was to build an image for each pull request and potentially allow new experiments to be triggered from pull requests. For this Github Actions are probably better to allow visibility into the build and push process.

**Comment by DonggeLiu** [comment]
> * [x]  Automatically push docker images from the PR to gcloud

An extended request: Only create & push image for changes in benchmark and source code. That is, no need to create images for documentation changes (e.g., #77).

**Comment by rishavtarway** [comment]
@DonggeLiu Can i work on the extended request you asked without creating images for documentation changes ?

**Comment by DonggeLiu** [comment]
Sure, @rishavtarway . Thanks for offering to help.
Do you have a plan on this?

**Comment by rishavtarway** [comment]
As of now i am going through all the last changes to have a better understanding of the extended request will surely update for the plan as soon as possible ?
